HPXMLtoOpenStudio/resources/hpxml_schematron/EPvalidator.xml

```diff
@@ -93,6 +94,15 @@
     </sch:rule>
   </sch:pattern>
 
+  <sch:pattern>
+    <sch:title>[Address]</sch:title>
+    <sch:rule context='/h:HPXML/h:Building/h:Site/h:Address'>
+      <sch:assert role='ERROR' test='count(h:StateCode) &lt;= 1'>Expected 0 or 1 element(s) for xpath: StateCode</sch:assert>
+      <sch:assert role='ERROR' test='h:StateCode[text()="AK" or text()="AL" or text()="AR" or text()="AZ" or text()="CA" or text()="CO" or text()="CT" or text()="DC" or text()="DE" or text()="FL" or text()="GA" or text()="HI" or text()="IA" or text()="ID" or text()="IL" or text()="IN" or text()="KS" or text()="KY" or text()="LA" or text()="MA" or text()="MD" or text()="ME" or text()="MI" or text()="MN" or text()="MO" or text()="MS" or text()="MT" or text()="NC" or text()="ND" or text()="NE" or text()="NH" or text()="NJ" or text()="NM" or text()="NV" or text()="NY" or text()="OH" or text()="OK" or text()="OR" or text()="PA" or text()="RI" or text()="SC" or text()="SD" or text()="TN" or text()="TX" or text()="UT" or text()="VA" or text()="VT" or text()="WA" or text()="WI" or text()="WV" or text()="WY"] or not(h:StateCode)'>Expected StateCode to be 'AK' or 'AL' or 'AR' or 'AZ' or 'CA' or 'CO' or 'CT' or 'DC' or 'DE' or 'FL' or 'GA' or 'HI' or 'IA' or 'ID' or 'IL' or 'IN' or 'KS' or 'KY' or 'LA' or 'MA' or 'MD' or 'ME' or 'MI' or 'MN' or 'MO' or 'MS' or 'MI' or 'NC' or 'ND' or 'NE' or 'NH' or 'NJ' or 'NM' or 'NV' or 'NY' or 'OH' or 'OK' or 'OR' or 'PA' or 'RI' or 'SC' or 'SD' or 'TN' or 'TX' or 'UT' or 'VA' or 'VT' or 'WA' or 'WI' or 'WV' or 'WY'</sch:assert>
```

Comment:
I would prefer that we not restrict these values, since users from another country (Canada, etc.) might want to provide a value that isn't a US state.

Reply:
![image](https://user-images.githubusercontent.com/8516790/150851024-03131e8f-5456-4cd4-925c-cd1ec12c4714.png)

Reply:
"can include"

Reply:
Also, the EPvalidator proposal doesn't allow Guam or some of these other locations.

Reply:
I removed it.